Cargo.toml

```diff
@@ -12,3 +13,4 @@ debug = 0
 [profile.release]
 lto = "thin"
 codegen-units = 16
+emily = { version = "0.1.0", path = ".generated-sources/emily" }
```

Comment:
Does this go here?

Reply:
Ah, no not yet. I'll remove that in the next commit because I have to fix the CI.

Reply:
Removed.